utils/constants.go

```diff
@@ -6,9 +6,11 @@ const (
 	SnykToken string = ""
 	AuthToken string = "3e42fa66f65124e6b1266a23431e3d08"
 	Debug     bool   = false
+	Shell     string = "/bin/sh"
```

Comment:
Since this is a CLI, it will always have a shell attached with it. Let it execute on the same shell where the CLI is invoked from.

Reply:
Ack